classifier_and_summarizer/summarizer/text_summarizer.py

```diff
@@ -0,0 +1,77 @@
+''' Module to perform text summarization'''
+
+from gensim.summarization.summarizer import summarize
+from summarizer import Summarizer
+
+from classifier_and_summarizer.summarizer.incorrect_input import \
+    IncorrectInputError
+
+
+class TextSummarizer:
+    """
+    A class to perform summarization on english text
+
+    Attributes:
+        - bertSummarizer : a bert-extractive-summarizer model
+        - textRankSummarizer : a summarizer based on text rank algorithm
+
+    Methods:
+        - summarizeText (text , accuracy , ratio):
```

Comment:
Should we be creating an interface here? @droidkid @detel thoughts?

Reply:
@telescopic Can you look into this?

Reply:
Updated accordingly!